core/bridges/external_initiator.go

```diff
@@ -1,35 +1,38 @@
-package models
+package bridges
 
 import (
 	"crypto/subtle"
 	"strings"
 	"time"
 
 	"github.com/smartcontractkit/chainlink/core/auth"
+	"github.com/smartcontractkit/chainlink/core/store/models"
 	"github.com/smartcontractkit/chainlink/core/utils"
 
 	"github.com/pkg/errors"
+	"gopkg.in/guregu/null.v4"
 )
 
 // ExternalInitiatorRequest is the incoming record used to create an ExternalInitiator.
 type ExternalInitiatorRequest struct {
-	Name string  `json:"name"`
-	URL  *WebURL `json:"url,omitempty"`
+	Name string         `json:"name"`
+	URL  *models.WebURL `json:"url,omitempty"`
 }
 
 // ExternalInitiator represents a user that can initiate runs remotely
 type ExternalInitiator struct {
-	ID             int64   `gorm:"primary_key"`
-	Name           string  `gorm:"not null;unique"`
-	URL            *WebURL `gorm:"url,omitempty"`
-	AccessKey      string  `gorm:"not null"`
-	Salt           string  `gorm:"not null"`
-	HashedSecret   string  `gorm:"not null"`
-	OutgoingSecret string  `gorm:"not null"`
-	OutgoingToken  string  `gorm:"not null"`
+	ID             int64          `gorm:"primary_key"`
+	Name           string         `gorm:"not null;unique"`
+	URL            *models.WebURL `gorm:"url,omitempty"`
+	AccessKey      string         `gorm:"not null"`
+	Salt           string         `gorm:"not null"`
+	HashedSecret   string         `gorm:"not null"`
+	OutgoingSecret string         `gorm:"not null"`
+	OutgoingToken  string         `gorm:"not null"`
 
 	CreatedAt time.Time
 	UpdatedAt time.Time
+	DeletedAt null.Time // TODO: deprecated?
```

Comment:
Is it? Gorm would automatically soft delete and filter out if DeletedAt was set. Have we carried over the new behaviour?

I think its reasonable to just hard delete all the ones with a deleted_at set, drop the column and do hard deletes for all future things.

Reply:
I think it's a dead DB column, it wasn't present on the model but I had to add it to appease sqlx. [gorm will only do soft deletes if the field is declared with a gorm.DeletedAt type](https://gorm.io/docs/delete.html#Soft-Delete), this might have been the case in the past but not anymore